### PR TITLE
Remove misplaced intro text in /doc + minor fixes

### PR DESCRIPTION
**Pull request by merland**
As outlined in issue #14639, there are some old and redundant 'running' instructions in two READMEs in /doc. This PR fixes that by instead linking to the official docs about running a node. 
 
Also, changed the download link to point to the download page instead of the releases page.

This is a near-duplicate of [PR14755](https://github.com/bitcoin/bitcoin/pull/14755) that was closed (maybe prematurely).  Please see that PR for some discussion.

### CODE REVIEW

**Comment by laanwj** [comment]
Tend to NACK.
I prefer having this documentation self-contained and not refer to bitcoin.org.
Yes, mentioning the full-node guide might be useful, but it shouldn't be the sole source of information.

**Comment by merland** [comment]
@laanwj  Thanks for reviewing, good to finally get some comments about this :)
It sounds reasonable to keep some minimal self-contained documentation, I updated the commit.  

There is also the question about some of text being misplaced. Phrases like "Bitcoin is a free open source peer-to-peer electronic cash system..." and "Bitcoin Core is the original Bitcoin client..." seem to be misplaced at this level in the tree. I think they are an heritage from before QT was merged upstream. These intro/setup texts are still removed in the updated commit.

After this, `README_windows.txt` contains only duplication, but removal causes Gitian build failure. This should be adressed but maybe in another PR.

Please re-review.

File: doc/README.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ Setup
 ---------------------
 Bitcoin Core is the original Bitcoin client and it builds the backbone of the network. It downloads and, by default, stores the entire history of Bitcoin transactions, which requires a few hundred gigabytes of disk space. Depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to a day or more.
 
-To download Bitcoin Core, visit [bitcoincore.org](https://bitcoincore.org/en/releases/).
+To download Bitcoin Core, visit [bitcoincore.org](https://bitcoincore.org/en/download/).
 
 Running
 ---------------------
-The following are some helpful notes on how to run Bitcoin Core on your native platform.
+The following are some helpful notes on how to run Bitcoin Core on your native platform. (For further details, please refer to https://bitcoin.org/en/full-node)
```
[the remark this answers is not in the document]
Comment (merland) [reply doc/README.md]: Thanks, I updated the commit.

**Comment by merland** [comment]
@laanwj I agree about the contradiction. I will try to look into that and hopefully submit another PR. 
Regarding this one, have you had a chance to look at the the latest version? 
Or anyone, of course.

**Pull request closed without merging**